guby_backend/rocketchat_utils.py

```diff
@@ -0,0 +1,34 @@
+from rocketchat_API.rocketchat import RocketChat
```

Comment:
Do you think we can make the name simpler by calling it `rocketchat.py`?

Reply:
Did not want to confuse with the actual package, but can do it. @raghothams  what do you think?

Reply:
Ah. That is also called `RocketChat`

Then how about a package called `utils` which I'm sure will have more. That way we can use `utils.rocketchat`. Cleaner perhaps?

Reply:
sure, we can create utils. But, I doubt if this is "util".

Or we can think of it as an abstraction & call it `gubychat.py`. Today the provider is RocketChat.

Reply:
+1 good point. I vouch for simply going with `chat.py` in that case. And dropping the extra qualifier since this is anyway under `guby_backend`. 

We can think about properly implementing the abstraction later.